### DISCUSSION TITLE
Stable can read RGBA but Lazer cannot

### DISCUSSION THREAD

**Discussion by papertek** [ppy/osu#20391] [Q&A] [answered]
This is in the skin.ini for the skin I'm using. When removing the alpha value the slider body works as expected in lazer
![image](https://user-images.githubusercontent.com/72106503/191426315-a5419ea5-c336-4a1d-9dbf-4902df18f938.png)

Expected slider body color in stable
![image](https://user-images.githubusercontent.com/72106503/191426401-f4fc80ff-04d5-485d-8ffd-f49c3fd5c5f3.png)

Slider body color in lazer
![image](https://user-images.githubusercontent.com/72106503/191426425-f7f4a71c-0800-47c5-8713-7ca81fcb729c.png)

I'm also noticing transparency with the hitcircle being wrong in lazer but i dunno if thats an editor thing

**Comment by peppy** [comment, marked as answer]
I'm pretty sure stable is just rendering ignoring the alpha portion.

You can fix this by removing the alpha part completely from your skin (`,0.78`).

I'll look into whether we can also ignore it on lazer for parity.
